test/authorization_tests.js

```diff
@@ -1,28 +1,51 @@
-'use strict';
-
-var async = require('async');
-var should = require('should');
-var assert = require('assert');
-var persona = require('../index.js');
-var _getOAuthToken = require('./utils')._getOAuthToken;
-var cryptojs = require('crypto-js');
-var sinon = require('sinon');
-var _ = require("lodash");
-
-describe("Persona Client Test Suite - Authorization Tests", function(){
-    describe("- Request authorization tests",function(){
-        it("should throw an error if guid is not present", function(done) {
-            var personaClient = persona.createClient({
-                persona_host:"persona",
-                persona_port:80,
-                persona_scheme:"http",
-                persona_oauth_route:"/oauth/tokens/",
-                redis_host:"localhost",
-                redis_port:6379,
-                redis_db:0,
-                enable_debug: false
-            });
+"use strict";
+
+var async = require("async");
+var should = require("should");
+var assert = require("assert");
+var persona = require("../index.js");
+var _getOAuthToken = require("./utils")._getOAuthToken;
+var sinon = require("sinon");
+var fs = require("fs");
+var nock = require("nock");
+
+describe("Persona Client Test Suite - Authorization Tests", function() {
+
+    var personaClient;
+
+    beforeEach(function recordHttpCalls() {
+        var testName = (this.currentTest.parent.title + " " + this.currentTest.title).replace(/ /g, "_");
+        nock.load(__dirname + "/responses/authorization/" + testName + ".json");
+        //nock.recorder.rec({
```

Comment:
should `nock.recorder` be abstracted out and called within a if statement `if (config.recording) { }`?